packages/@sanity/desk-tool/src/panes/document/documentPanel/header/DocumentPanelHeader.tsx

```diff
@@ -18,118 +18,125 @@ import {DocumentHeaderTitle} from './DocumentHeaderTitle'
 
 export interface DocumentPanelHeaderProps {
   rootElement: HTMLDivElement | null
+  isReferencedDocument?: boolean
 }
 
 const isActionButton = (item: PaneMenuItem) => Boolean(item.showAsAction)
 const isMenuButton = negate(isActionButton)
 
 export const DocumentPanelHeader = memo(
-  forwardRef(({rootElement}: DocumentPanelHeaderProps, ref: React.ForwardedRef<HTMLDivElement>) => {
-    const {
-      documentSchema,
-      handleMenuAction,
-      handlePaneClose,
-      handlePaneSplit,
-      historyController,
-      markers,
-      menuItems,
-      menuItemGroups,
-      ready,
-      views,
-    } = useDocumentPane()
-    const {revTime: rev} = historyController
-    const {features} = useDeskTool()
-    const {index, BackLink, hasGroupSiblings} = usePaneRouter()
-    const contextMenuItems = useMemo(() => menuItems.filter(isMenuButton), [menuItems])
-    const [isValidationOpen, setValidationOpen] = React.useState<boolean>(false)
-    const showTabs = views.length > 1
-    const showVersionMenu = features.reviewChanges
+  forwardRef(
```

Comment:
cant really see what changed below because the linting also changed. Is it possible to revert the changes not related to this PR?

Reply:
Prettier is "at fault" here - not much to do about it, I think. One alternative would be to refactor the code a bit so prettier doesn't wrap it further, heh